docs/changehistory/NextVersion.md

```diff
@@ -2,11 +2,34 @@
 
 Table of contents:
 
-- [@itwin/components-react](#itwincomponents-react)
+- [@itwin/appui-react](#itwinappui-react)
   - [Deprecations](#deprecations)
+  - [Additions](#additions)
+  - [Changes](#changes)
+- [@itwin/components-react](#itwincomponents-react)
+  - [Deprecations](#deprecations-1)
 - [@itwin/core-react](#itwincore-react)
   - [Fixes](#fixes)
 
+## @itwin/appui-react
+
+### Deprecations
+
+- Deprecated `StatusBar` component in favor of `StatusBarComposer`. [#848](https://github.com/iTwin/appui/pull/848)
+- Deprecated `StatusBar` namespace. Please use the components directly. [#848](https://github.com/iTwin/appui/pull/848)
+  - Deprecated `StatusBar.Popup` and `StatusBarPopup` in favor of `StatusBarPopover`.
+  - Deprecated `StatusBar.Field` and `StatusBarField` in favor of [iTwinUI Button](https://itwinui.bentley.com/docs/button).
+- Deprecated `StatusBarIndicator` in favor of [iTwinUI Button](https://itwinui.bentley.com/docs/button) (or other components) and AppUI `StatusBarPopup`. [#848](https://github.com/iTwin/appui/pull/848)
+- Deprecated `StatusBarLabelIndicator` in favor of [iTwinUI Label](https://itwinui.bentley.com/docs/typography#label) and AppUI `Icon`. [#848](https://github.com/iTwin/appui/pull/848)
+
+### Additions
+
+- Added `StatusBarPopover` to replace `StatusBar.Popup`. The component uses [iTwinUI Popover](https://itwinui.bentley.com/docs/popover) with some consistent styling for all `StatusBar` fields. `StatusBarPopover` should wrap the element that triggers it (e.g. a button). [#848](https://github.com/iTwin/appui/pull/848)
```

Comment:
Mention the addition, usage and use-cases of `StatusBarPopover.ExpandIndicator`.

Reply:
I mentioned it below already. I added some examples now